CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Upgrade pymongo to version 4
 - When updating disease terms, check that all terms are consistent with a DiseaseTerm model before dropping the old collection
 - Better separation between modules loading HPO terms and diseases
+- Stricter validation of mandatory genome build key when loading a case. Allowed values are ['37','38',37,38]
```

Comment:
Hm, why make it mandatory suddenly if we have had a default forever? I would not have minded it from the start - it is rather crucial info - but isn't the change sudden?